src/test/enhetstester/kotlin/no/nav/familie/ba/sak/kjerne/forrigebehandling/EndringIEndretUtbetalingAndelUtilTest.kt

```diff
@@ -0,0 +1,144 @@
+package no.nav.familie.ba.sak.kjerne.forrigebehandling
+
+import no.nav.familie.ba.sak.common.førsteDagIInneværendeMåned
+import no.nav.familie.ba.sak.common.lagEndretUtbetalingAndel
+import no.nav.familie.ba.sak.common.lagPerson
+import no.nav.familie.ba.sak.kjerne.endretutbetaling.domene.Årsak
+import no.nav.familie.ba.sak.kjerne.grunnlag.personopplysninger.PersonType
+import no.nav.familie.ba.sak.kjerne.tidslinje.tidspunkt.tilYearMonth
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.Test
+import java.math.BigDecimal
+import java.time.YearMonth
+
+class EndringIEndretUtbetalingAndelUtilTest {
+
+    val jan22 = YearMonth.of(2022, 1)
+    val feb22 = YearMonth.of(2022, 2)
+    val mai22 = YearMonth.of(2022, 5)
+    val aug22 = YearMonth.of(2022, 8)
+    val des22 = YearMonth.of(2022, 12)
+
+    @Test
+    fun `Endring i endret utbetaling andel - skal returnere true hvis årsak er endret`() {
```

Comment:
```suggestion
    fun `Endring i endret utbetaling andel - skal returnere endret periode hvis årsak er endret`() {
```
Er det mer riktig navngiving av test hvis vi bytter fra true til endret periode?

Reply:
✅

Reply:
✅